.github/workflows/update_prebuilts.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         uses: actions/checkout@v2
       - name: "Run update prebuilt snapshot ids script"
         shell: bash
-        run: development/update_playground.sh
+        run: development/update_playground.sh a
```

Comment:
typo?

Reply:
https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:development/update_playground.sh;l=25

Reply:
Oh I forgot I wrote this - we definitely don't want to update metalava to ToT all the time because we need to match whatever version we're using in AOSP.

Reply:
The correct way to fix this is actually to add this into importMaven so that it automatically gets bumped when we do it in AOSP like here: aosp/2619889

If we bump to ToT, we will get ahead of AOSP and have API file conflicts

Reply:
ok that is reasonable, i'll give it a try and remove the `a` change from here.